packages/components/src/components/Select/SelectPopover.tsx

```diff
@@ -22,6 +23,7 @@ const SelectPopover = React.forwardRef<HTMLDivElement, SelectPopoverProps>(
         flexDirection="column"
         gutter={gutter}
         maxHeight="250px"
+        maxW="400px"
```

Comment:
Done: https://github.com/Localitos/pluto/pull/1741/commits/9819bd11035415f0d9f57e4e37b65f9f82e7fbd9#diff-6cb2174762b1b0bc94179bf9bbc74e85c0b30de0dfbf459b89fb92ee02fba862R65
I think it might be useful. 🙏🏻